.codegen/model.go.tmpl

```diff
@@ -31,7 +31,7 @@ type {{.PascalName}} struct {
 {{end}}
 
 {{- define "field-tag" -}}
-  {{if .IsJson}}tfsdk:"{{if and (ne .Entity.Terraform nil) (ne .Entity.Terraform.Alias "") }}{{.Entity.Terraform.Alias}}{{else}}{{.Name}}{{end}}" tf:"{{- $first := true -}}{{- if not .Required -}}{{- if not $first -}},{{end}}optional{{- $first = false -}}{{- end -}}{{- if .Entity.IsObject -}}{{- if not $first -}},{{end}}object{{- $first = false -}}{{- end -}}"{{else}}tfsdk:"-"{{end -}}
+  {{if .IsJson}}tfsdk:"{{if and (ne .Entity.Terraform nil) (ne .Entity.Terraform.Alias "") }}{{.Entity.Terraform.Alias}}{{else}}{{.Name}}{{end}}" tf:"{{.TfOptions}}"{{else}}tfsdk:"-"{{end -}}
```

Comment:
@rauchy I noticed that you merged this [PR](https://github.com/databricks/terraform-provider-databricks/commit/1f59bfdad60cb5e8c16ad24d054bc15f3cb309d9) here, but do you think you can follow this style I'm proposing to move the heavy logics into the genkit file in universe? - related [PR](https://github.com/databricks/universe/pull/755258). That way the template will be very easy to follow